Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -181,10 +182,19 @@ namespace ROS2
             auto collidersNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), false, true);
             auto visualNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), true, false);
 
+            AZ::Crc32 params_crc;
+            for (auto& [key, value] : m_params)
+            {
+                params_crc.Add(key);
+                params_crc.Add(value);
+            }
+
+            auto dir_suffix = AZStd::string::format("%d", AZ::u32(params_crc));
+
             if (m_importAssetWithUrdf)
             {
                 m_urdfAssetsMapping = AZStd::make_shared<Utils::UrdfAssetMap>(
-                    Utils::CopyAssetForURDFAndCreateAssetMap(m_meshNames, m_urdfPath.String(), collidersNames, visualNames));
+                    Utils::CopyAssetForURDFAndCreateAssetMap(m_meshNames, m_urdfPath.String(), collidersNames, visualNames, dir_suffix));
```

Comment:
The naming scheme of the `dir_suffix` parameter is inconsistent with the other parameters and should be camel case per the [coding standard](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#naming).

i.e the name could be `dirSuffix`